### PR TITLE
added option for sensor MAX6675

### PR DESCRIPTION
**Pull request by danielkucera**

### CODE REVIEW

File: src/ControllerBase.h
Comment (foxis) [inline src/ControllerBase.h]: It is much more preferable to have explicit defines for each sensor instead of specifying the chip version, e.g.:
```
#define TEMPERATURE_SENSOR_MAX31855
//#define TEMPERATURE_SENSOR_MAX6675
```

Also, I think it is much better to specify these defines in the `platformio.ini` file so that one can specify different targets there.

**Review by foxis — CHANGES_REQUESTED** [review]
Thank you very much for this PR!
I'll merge it once the defines are formatted as suggested and actual sensor specification is moved to `platformio.ini` file.

File: src/ControllerBase.h
Comment (danielkucera) [reply src/ControllerBase.h]: done. please recheck.

File: platformio.ini
Comment (foxis) [inline platformio.ini]: One last thing is left to do:

There needs to be a `[platformio]` section with `env_default` field.
Also please add a second env section named e.g. `env:reflow_{name of MAX chip}` and copy your changes there.
That way there will be two environments, one for MAX31855 and one for MAX6675 and they will be using the flags/dependencies that are needed only for that build environment.
That is helpful, because one wouldn't need to edit `platformio.ini` every time they want to build against different chips and also it would be very clear what settings are needed for what chip.

Also, since this project also builds with Wemoss D1 mini, I'll add an environment for that one too after this change lands.

One more great thing of having multiple build environments - automated builds that I'm planning to configure after I finish a blog article about this project :)

**Review by foxis — CHANGES_REQUESTED** [review]
Looks great. One small change to the platformio environment configuration and it's good to go.

File: platformio.ini
Comment (danielkucera) [reply platformio.ini]: done.

**Comment by danielkucera** [comment]
Ping.

**Pull request merged**
